### PR TITLE
Updates CI tooling

### PR DESCRIPTION
**Pull request by helrond**
- Removes some old detritus from container deployments
- uses common deploy scripts
- updates port settings
- updates requirements

### CODE REVIEW

**Review by bonniegee — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -9,21 +9,22 @@ env:
   global:
     - CONTAINER: request-broker-web
     - APPLICATION_NAME: request_broker
-    - DJANGO_PORT: 8000
+    - APPLICATION_PORT: 80
```
Comment (HaSistrunk) [inline .travis.yml:12]: Should this be 8000?

**Review by HaSistrunk — COMMENTED** [review]
Do you need the `wait_until_deployed: false` in travis.yml?

File: .travis.yml (hunk 1 above)
Comment (HaSistrunk) [inline .travis.yml:12]: Should this be 8000? If not, can you explain why the docker compose file is using `- "${APPLICATION_PORT:-8000}:${APPLICATION_PORT:-8000}"`?

**Comment by helrond** [comment]
> Do you need the `wait_until_deployed: false` in travis.yml?

The `wait_until_deployed` value is `false` by default, see [Travis CI Documentation](https://docs.travis-ci.com/user/deployment/codedeploy/#waiting-for-deployments)

File: .travis.yml (hunk 1 above)
Comment (helrond) [reply .travis.yml:12]: On the server, we want to deploy the application on port 80. Locally (in our container) we want it to run on port 8000.

When you run the local container, the environment variables in the Travis config file (and whatever is entered in the Travis UI) don't enter into the picture, so the values in the Compose file are used. When deploying, the Travis variables override everything else, which means that port 80 gets used for the deployment artifacts.

Comment (HaSistrunk) [reply .travis.yml:12]: Makes sense, thank!

**Comment by HaSistrunk** [comment]
> > Do you need the `wait_until_deployed: false` in travis.yml?
> 
> The `wait_until_deployed` value is `false` by default, see [Travis CI Documentation](https://docs.travis-ci.com/user/deployment/codedeploy/#waiting-for-deployments)

Then does it make sense to remove that from line 40 of the .travis.yml file?

**Comment by helrond** [comment]
@HaSistrunk ah, sorry about that! Yes, got rid of it...

**Review by HaSistrunk — APPROVED** [review]

**Review by p-galligan — APPROVED** [review]

**Pull request merged**